admin/resources/js/components/searchRequest/SearchRequestTable.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useMemo } from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { Button } from "@common/components";
+import { notEmpty } from "@common/helpers/util";
+import { navigate, useLocation } from "@common/helpers/router";
+import { commonMessages } from "@common/messages";
+import { FromArray } from "@common/types/utilityTypes";
+import {
+  GetPoolCandidateSearchRequestsQuery,
+  useGetPoolCandidateSearchRequestsQuery,
+} from "../../api/generated";
+import Table, { ColumnsOf } from "../Table";
+import DashboardContentContainer from "../DashboardContentContainer";
+import { getLocale } from "@common/helpers/localize";
+
+type Data = NonNullable<FromArray<GetPoolCandidateSearchRequestsQuery["poolCandidateSearchRequests"]>>;
+
+export const SearchRequestTable: React.FunctionComponent<GetPoolCandidateSearchRequestsQuery & { editUrlRoot: string }> = ({ poolCandidateSearchRequests, editUrlRoot }) => {
+ const intl = useIntl();
+ const locale = getLocale(intl);
+ const columns = useMemo<ColumnsOf<Data>>(
+   () => [
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "ID",
+        description: "Title displayed on the search request table id column.",
+      }),
+       accessor: "id"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Date Received",
+        description: "Title displayed on the search request table requested date column.",
+      }),
+       accessor: "requestedDate"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Status",
+        description: "Title displayed on the search request table status column.",
+      }),
+       accessor: "status"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Manager",
+        description: "Title displayed on the search request table first name column.",
+      }),
+       accessor: "fullName"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Department",
+        description: "Title displayed on the search request table department column.",
+      }),
+       accessor: ({ department }) => department?.name?.[locale]
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Email",
+        description: "Title displayed on the search request table email column.",
+      }),
+       accessor: "email"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Job Title",
+        description: "Title displayed on the search request table job title column.",
+      }),
+       accessor: "jobTitle"
+     },
+     {
+       Header: intl.formatMessage({
+        defaultMessage: "Pool",
+        description: "Title displayed on the search request table pool column.",
+      }),
+       accessor: ({ poolCandidateFilter }) => poolCandidateFilter.id
```

Comment:
Is this meant to be the name (in EN or FR) of the Pool?

Reply:
Yea it is